Automattic/wp-calypso
# Fixes the loading screen for the Advertising details page

Related to SELFSERVE-616

## Proposed Changes

This PR fixes the loading skeleton for the campaign details page. We have a strange movement when the page is loading, this happened because of the loading page don't using the section page name in the body.

**Before (Notice that the page moves to the right after its finished, and the background changes from grey to white):**
![2023-06-23 10 21 47](https://github.com/Automattic/wp-calypso/assets/1258162/414fccd2-6eda-4073-9dba-d5e06cdc12af)

**After:**
![2023-06-23 10 23 33](https://github.com/Automattic/wp-calypso/assets/1258162/765b2723-3334-4e89-b052-f6c8ce705baa)


## Testing Instructions



* Open the live preview link and navigate to Tools->Advertising
* Click on Campaigns in the navigation bar
* Click on any campaign
* Refresh the page.
* Verify that the correct background is shown when the page loads

## Pre-merge Checklist
